pkg/graph/resolvers/mto_milestone_test.go

```diff
@@ -619,7 +619,10 @@ func (suite *ResolverSuite) TestMTOMilestoneNoLinkedSolutions_MultiplePlans() {
 func (suite *ResolverSuite) TestMTOMilestoneAssignedTo() {
 	// Create model plan
 	plan := suite.createModelPlan("testing milestone assigned_to")
-	assignedTo := uuid.MustParse("00000001-0001-0001-0001-000000000001") // Mint user ID
+	collaborator := suite.createPlanCollaborator(plan, "CLAB", []models.TeamRole{models.TeamRoleLeadership})
```

Comment:
Test changes looks good here, nice to use the suite create functions instead of hard coded in uuids